react/features/toolbox/components/web/Toolbox.js

```diff
@@ -41,6 +44,9 @@ import {
     SettingsButton,
     openSettingsDialog
 } from '../../../settings';
+import {
```

Comment:
The imports should be sorted alphabetically first by the filename.

Reply:
Please address this comment

Reply:
Ordered alphabetically by filename